batch/batch/server/database.py

```diff
@@ -1,159 +1,133 @@
+import os
 import json
-import uuid
-import asyncio
-import aiomysql
-import pymysql
-from asyncinit import asyncinit
 
+from ..database import Database, Table
 
-def run_synchronous(coro):
-    loop = asyncio.get_event_loop()
-    return loop.run_until_complete(coro)
-
-
-@asyncinit
-class Database:
-    @staticmethod
-    def create_synchronous(config_file):
-        db = run_synchronous(Database(config_file))
-        return db
 
+class BatchDatabase(Database):
```

Comment:
It kind of feels like we should just use the parent's `run_synchronous` static method.

Reply:
I think it's fine the way it is for now.